tests/unit/dataactvalidator/test_b18_object_class_program_activity_1.py

```diff
@@ -0,0 +1,45 @@
+from factory import fuzzy
+from tests.unit.dataactcore.factories.staging import ObjectClassProgramActivityFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+
+_FILE = 'b18_object_class_program_activity_1'
+
+not_req_dict = {
+        "submission_id":fuzzy.FuzzyInteger(9999),
+        "object_class":fuzzy.FuzzyInteger(1000,9999),
+        "by_direct_reimbursable_fun":""
+}
+
+req_present_dict = {
+        "submission_id":fuzzy.FuzzyInteger(9999),
+        "object_class":fuzzy.FuzzyInteger(999),
+        "by_direct_reimbursable_fun":"d"
+}
+
+req_absent_dict = {
+        "submission_id":fuzzy.FuzzyInteger(9999),
+        "object_class":fuzzy.FuzzyInteger(999),
+        "by_direct_reimbursable_fun":""
+}
+
+def test_column_headers(database):
+    expected_subset = {"row_number", "object_class", "by_direct_reimbursable_fun"}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset <= actual
```

Comment:
I was basing this on the existing unit tests that check this as a subset, I'll ask if we expect the result to have any other columns or if this should be the complete result.